Leap-AI (Independent Publisher)

sekharmalla opened the pull request:
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

<img width="649" alt="image" src="https://user-images.githubusercontent.com/43902003/226176178-e704ed5b-28c9-47b2-ae26-812292756523.png">

sekharmalla committed:
Update README.md

sekharmalla committed:
Revert "Update README.md"

This reverts commit aa02b18291793635924d8e72d18993e7fea464f4.

sekharmalla committed:
Adding Leap AI Independent Publisher Connector

Amjed-Ayoub commented:
Hello @sekharmalla,

Thank you so much for submitting this PR. May I ask you please to double check your connector files? this PR has 0 files.

Thank you so much.

sekharmalla committed:
Update README.md

Updated README.MD with original content

sekharmalla commented:
Hello @Amjed-Ayoub,

Thanks for contacting me, it is strange to know that there no files in this PR coz I can clearly see them in the files changed. 

<img width="1273" alt="image" src="https://user-images.githubusercontent.com/43902003/226772408-1c6aadaf-4129-4833-8867-a01e26297ab9.png">

<img width="1429" alt="image" src="https://user-images.githubusercontent.com/43902003/226772548-cc234da9-b373-4092-9cea-d644de26b259.png">
Thank You

Amjed-Ayoub commented on independent-publisher-connectors/Leap-Ai/apiProperties.json:
```diff
@@ -0,0 +1,22 @@
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```
@sekharmalla for all our independent publishers we ask to set the "iconBrandColor" to "#da3b01". Please see out template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).

sekharmalla replied:
done that.

Amjed-Ayoub commented on independent-publisher-connectors/Leap-Ai/apiProperties.json:
```diff
@@ -0,0 +1,22 @@
+    },
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "publisher": "Chandra Sekhar Malla"
```
@sekharmalla please add the stackOwner property. Please see out template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).
"stackOwner": "Name of company or organization that connector is connecting to."

sekharmalla replied:
done that.

sekharmalla committed:
Merge branch 'microsoft:dev' into Leapai

sekharmalla committed:
Update README.md

Updated README.md as per Microsoft template

sekharmalla commented on independent-publisher-connectors/Leap-Ai/README.md:
```diff
@@ -0,0 +1,75 @@
+# Leap AI
```
Hi @Amjed-Ayoub  as per your request I have made some changes to the template. Hope that's okay

sekharmalla committed:
Update apiDefinition.swagger.json

changes as follows:

- removed API name from the definition file.
- changed all operationid to camel case.
- updated all response description to success.

sekharmalla committed:
Update apiProperties.json

changes as follows:

- added stack owner

sekharmalla commented on independent-publisher-connectors/Leap-Ai/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,857 @@
+  "swagger": "2.0",
+  "info": {
+    "title": "LeapAi AI",
+    "description": "API for LeapAi image recognition service.",
```
Done, fixed.

sekharmalla commented on independent-publisher-connectors/Leap-Ai/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,857 @@
+          }
+        },
+        "summary": "List All Models",
+        "operationId": "List_All_Models",
```
Done, changed all operationid to camel case

Amjed-Ayoub commented:
[[certify-connector]]

Amjed-Ayoub commented:
Hello @sekharmalla ,
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

Amjed-Ayoub commented:
Hello @sekharmalla ,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

Amjed-Ayoub reviewed (approved):
Hello @sekharmalla ,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program.
Thank you very much for working with us.
